src/Tests/Microsoft.NET.TestFramework/Commands/MSBuildCommand.cs

```diff
@@ -72,38 +72,53 @@ internal static string FindProjectFile(ref string projectRootPath, string relati
             return buildProjectFiles[0];
         }
 
-        public virtual DirectoryInfo GetOutputDirectory(string targetFramework, string configuration = "Debug", string runtimeIdentifier = "")
+        private DirectoryInfo GetDirectoryRelativeToProjectRoot(params string[] dirs)
+        {
+            string relativePath = Path.Combine(dirs);
+            string output = Path.Combine(ProjectRootPath, relativePath);
+            return new DirectoryInfo(output);
+        }
+
+        public virtual DirectoryInfo GetBaseOutputDirectory()
+        {
+            return GetDirectoryRelativeToProjectRoot("bin");
+        }
+
+        public virtual DirectoryInfo GetBaseIntermediateDirectory()
+        {
+            return GetDirectoryRelativeToProjectRoot("obj");
+        }
+
+        public virtual DirectoryInfo GetNonSDKOutputDirectory(string configuration = "Debug")
```

Comment:
It doesn't look like these new "GetNonSdk" methods are called anywhere.  Is there a reason to add them?

Reply:
Yes, to test non-sdk style projects.

Reply:
Are there any tests which use these methods though?  I didn't see any.